webserver/pkgpkr/webservice/recommender_service.py

```diff
@@ -64,7 +64,8 @@ def get_recommendations(self, dependencies):
                     b.bounded_popularity,
                     s.bounded_similarity,
                     b.categories,
-                    b.display_date
+                    b.display_date,
```

Comment:
Do we need to return the date anymore if we're not displaying it?

Reply:
We still display it as a tooltip